po/pl.po

```diff
@@ -529,3 +533,9 @@ msgstr "Wybór urządzenia"
 #: data/ui/Window.ui:8
 msgid "_About Piper"
 msgstr "_O programie"
+
+msgid ""
```

Comment:
Isn't here `#: piper/window.py:55` missing? Anyway, you can rebase against master now since it's merged.

Reply:
It doesn’t matter, the .po files will get updated during build anyway.

Reply:
OK, rebased!

Reply:
Thanks, didn't know that.